astropy/conftest.py

```diff
@@ -103,6 +103,21 @@ def pytest_configure(config):
     PYTEST_HEADER_MODULES["asdf-astropy"] = "asdf_astropy"
     TESTED_VERSIONS["Astropy"] = __version__
 
+    # Limit the number of threads used by each worker when pytest-xdist is in
+    # use.  Lifted from https://github.com/scipy/scipy/pull/14441
+    # and https://github.com/scikit-learn/scikit-learn/pull/25918
+    try:
+        from threadpoolctl import threadpool_limits
+
```

Comment:
It is good practice to have as few lines inside a `try`-block as possible. In this case
```suggestion
    try:
        from threadpoolctl import threadpool_limits
    except ImportError:
        pass
    else:
        ...
```

Reply:
Thanks!  Fixed.